src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -99,8 +219,8 @@ const useCoreChatSystemHook = () => {
   };
 
   const helperDetermineIsMyChildComment = (postNotification: AnonymousPostNotification) => {
-    let latestComment: Comment = null;
-    let latestCommentLatestChild: LatestChildrenComment = null;
+    let latestComment: Comment | null = null;
+    let latestCommentLatestChild: LatestChildrenComment | null = null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variables `latestComment` and `latestCommentLatestChild` are initialized as `null` but never used in the provided code snippet. If these variables are not used later in the code, consider removing them to improve readability and maintainability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->